apps/src/music/views/Timeline.tsx

```diff
@@ -111,9 +112,60 @@ const Timeline: React.FunctionComponent<TimelineProps> = ({
     return eventHeight > 8 ? 3 : eventHeight > 6 ? 2 : 1;
   }, []);
 
+  const timelineElements = Array.from(
+    document.querySelectorAll<HTMLElement>('#timeline-element')
+  );
+
+  const onKeyDown = useCallback(
+    (event: React.KeyboardEvent<HTMLButtonElement>) => {
+      const currentIndex = timelineElements.indexOf(event.currentTarget);
+
+      if (event.key === 'ArrowRight' || event.key === 'ArrowDown') {
+        // Prevent scroll action & move focus to the next element
+        event.preventDefault();
+        const nextIndex = currentIndex + 1;
+        if (nextIndex < timelineElements.length) {
```

Comment:
for my understanding: is it better to loop around when you hit the end or to have nothing happen if you are at the end and try to arrow key to the next element?

Reply:
Hmm @dju90 what do you think? On one hand, the arrows 'doing nothing' are a helpful indication that you're at the beginning/end of the song. On the other, in a broader tab-nav sense, users are used to looping back around.